packages/sveltekit/src/config/withSentryViteConfig.ts

```diff
@@ -0,0 +1,55 @@
+import { UserConfig, UserConfigExport } from 'vite';
+import { injectSentryInitPlugin } from './vitePlugins';
+
+/**
+ * This function adds Sentry-specific configuration to your Vite config.
+ * Pass your config to this function and make sure the return value is exported
+ * from your `vite.config.js` file.
+ *
+ * Note: If you're already wrapping your config with another wrapper,
+ * for instance with `defineConfig` from vitest, make sure
+ * that the Sentry wrapper is the outermost one.
```

Comment:
Hmm contrary to my initial testing, this doesn't seem to matter with the vitest wrapper. Should we nevertheless leave it in for a general reminder?

Reply:
Yes, because this can always change in the future where we need this behavior. Better to mandate it for now.

Reply:
Yup, agreed, that was my thought as well